tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -221,8 +221,11 @@ generate_qemu_options() {
 	fi
 
 	# Disabled options
-
-	if gt_eq "${qemu_version}" "5.0.0" ; then
+	if gt_eq "${qemu_version}" "6.0.0" ; then
+		# Disable block migration in the main migration stream
```

Comment:
Could you explain why the option is now necessary in the comment?

Reply:
It isn't - that's just copied from the existing constraints for 5.0.0.